Gemfile

```diff
@@ -156,6 +155,7 @@ group :development do
   # misc
   gem "fix-db-schema-conflicts", "~> 2.0.0"
   gem "letter_opener", "~> 1.4.1"
+  # gem "rubocop", "~> 0.52.1", require: false
```

Comment:
Curious why this is commented. Can we just delete this, or if you think we should leave it, can you put a comment as to why we have a commented line in the gemfile?

Reply:
Wasn't supposed to be commented, I think I had been troubleshooting something with the gem installation process with the axlsx update and I forgot to put it back in, it's supposed to be there.